tests/tests_pytorch/helpers/runif.py

```diff
@@ -176,8 +176,10 @@ def __new__(
             reasons.append("unimplemented on Windows")
 
         if tpu:
-            conditions.append(not _TPU_AVAILABLE)
+            env_flag = os.getenv("PL_RUN_TPU_TESTS", "0")
+            conditions.append(env_flag != "1" or not _TPU_AVAILABLE)
```

Comment:
I suggest you don't add the env flag check, for the same reason as https://github.com/Lightning-AI/lightning/pull/13676

```suggestion
            conditions.append(not _TPU_AVAILABLE)
```